service/src/main/java/bio/terra/cbas/dependencies/wes/CromwellClient.java

```diff
@@ -19,22 +17,18 @@ public class CromwellClient {
 
   private final CromwellServerConfiguration cromwellServerConfiguration;
   private final DependencyUrlLoader dependencyUrlLoader;
-  private final CredentialLoader credentialLoader;
 
   private final OkHttpClient singletonHttpClient;
 
   public CromwellClient(
       CromwellServerConfiguration cromwellServerConfiguration,
-      DependencyUrlLoader dependencyUrlLoader,
-      CredentialLoader credentialLoader) {
+      DependencyUrlLoader dependencyUrlLoader) {
     this.cromwellServerConfiguration = cromwellServerConfiguration;
     this.dependencyUrlLoader = dependencyUrlLoader;
-    this.credentialLoader = credentialLoader;
     singletonHttpClient = new ApiClient().getHttpClient();
   }
 
-  public ApiClient getWriteApiClient()
-      throws DependencyNotAvailableException, AzureAccessTokenException {
+  public ApiClient getWriteApiClient(String accessToken) throws DependencyNotAvailableException {
```

Comment:
Is this still needed or is it replaced by the `cromwellWriteClient` in CromwellService?

Reply:
Nvm, I see that this is used in the bean creation.